LICENSE.md

```diff
@@ -1,10 +1,7 @@
 Copyright VMware, Inc.  
 SPDX-License-Identifier: APACHE-2.0
 
-This product includes third-party open source software.
-Additional copyright and licensing information can be found inside the
-Bitnami packages themselves and/or at https://bitnami.com/open-source
-
+This product includes third-party open source software. Additional copyright and licensing information can be found inside the Bitnami packages themselves and/or at https://bitnami.com/open-source
 ----
```

Comment:
Shouldn't we add here the blank line as we're doing in https://github.com/bitnami/vms/pull/987 and https://github.com/bitnami/charts/pull/17134?
```suggestion
This product includes third-party open source software. Additional copyright and licensing information can be found inside the Bitnami packages themselves and/or at https://bitnami.com/open-source

----
```